docs/Splashkit/Modules/Machine Learning/SRS.md

```diff
@@ -49,69 +71,58 @@ This system has a few assumptions. These include:
 Each of these assumptions is important for the use and requirements of the system. The system should
 be able to deal with multiple move requests in quick succession.
 
-## System Requirements
+## 3. System Features and Requirements
 
-## Functional Requirements
+### 3.1 Functional Requirements
 
 The functional requirements of the system are as follows:
 
 - The system should be able to accept a given Game wrapper for training or initialisation.
 - The system should be able to accept a different Game wrapper for playing.
 - The system should return readable moves for any given game state to the user whenever requested.
 
-## Interface Requirements
+### 3.2 External Interface Requirements
 
-The interface for the system will be entirely within the Splashkit platform. As such, it will have
-the following requirements:
+The interface for the system will be entirely within the Splashkit platform. The system will 
+provide API functions as its external interface. As such, it will have the following 
+requirements:
 
 - The system's interface should be following the same format and design as other sections of the
-  Splashkit Platform.
+  SplashKit Platform.
 - The system should be easy to use.
 - The system should return results in a readable way.
 
-## Hardware Interfaces
-
-This project is completely software based.
-
-## Communication Interfaces
+#### **Communication Interfaces**
 
 Documentation is to be provided so that the user can know how to use the system.
 
-## Software Interfaces
+#### **Software Interfaces**
 
 The Machine Learning Module includes the following components:
 
 - Game wrapper: An API for the user to implement so that an Agent can make moves
 - Agents: An Agent will be chosen by the user that works best with the given Game.
+- ANN API: An API allowing the user to create and train a Neural Network.
 
-## System Features
+### 3.3 System Features
 
 The system mainly focuses on the backend assistance for the user. As such, the features of the
 system are as follows:
 
-- The system will accept a game format to so that game states can be read and moves can be made by
+- The system will accept a game format so that game states can be read and moves can be made by
   the AI.
 - The system can accept new game states and analyses the game state to produce a move.
 - The system will return a move vector containing the information about the move it 
 wishes to play so that the game state can be updated. This move vector is to be processed 
 by the user.
 
-## Non-functional requirements.
+### 3.4 Nonfunctional Requirements
```

Comment:
Nonfunctional requirements should include the testing of the functional requirements, with info on who tests and what tests will be run. These just need to be added and then this section is all good

Reply:
not sure how to fit who tests and what tests need to be run into the SRS, but since it is already in the Testing Strategy, I was still a bit vague. Let me know if [this](https://github.com/thoth-tech/documentation/pull/101/commits/4ad7f308ea0b9ac7da0735af824c0dbb4d926816) is ok!

Reply:
It doesn't have to be in depth about the testing, just needs to be like:

Testing on {insert functional requirement] will be completed by [insert team] to determine if it passes quality assurance.

As our team is small, and there is no direct testing team, the testing will be completed by people within Modules.